### PR TITLE
Motor and preset tabs android css fixes

### PR DESCRIPTION
**Pull request by Asizon**
Fixes telemetry box values and presets dialog padding for android.

motortab
presetsdialog

### CODE REVIEW

**Review by McGiverGim — DISMISSED** [review]

**Comment by Asizon** [comment]
Sorry @McGiverGim ,we have found that this "not" css function doesnt work on chrome versions < 88 so we change it.
Thanks @limonspb to get to the end of the issue.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: src/tabs/presets/DetailedDialog/PresetsDetailedDialog.css
Comment (haslinghuis) [inline src/tabs/presets/DetailedDialog/PresetsDetailedDialog.css:113]: Please add newline here?

Comment (Asizon) [reply src/tabs/presets/DetailedDialog/PresetsDetailedDialog.css:113]: @haslinghuis whereee?

Comment (Asizon) [reply src/tabs/presets/DetailedDialog/PresetsDetailedDialog.css]: done

**Review by haslinghuis — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
